app/questionnaire/questionnaire_schema.py

```diff
@@ -94,11 +95,37 @@ def __init__(
         self._populate_answer_dependencies()
         self._populate_when_rules_section_dependencies()
         self._populate_calculated_summary_section_dependencies()
+        self._populate_min_max_map()
```

Comment:
Maybe name it, `populate_numeric_ranges` or `populate_mix_max_for_numeric_answers`?